girder/api/rest.py

```diff
@@ -345,6 +368,10 @@ def wrapped(*args, **kwargs):
                     raise RestException(
                         'Invalid %s id (%s).' % (model.name, str(id)))
 
+                if self.requiredFlags and not self.force:
```

Comment:
Thanks for adding those other tests.  Can you add a test for the case where you require flags for the loadmodel decoration and also set force?  It is kind of a degenerate case, because the client would be saying "require this specific level of access, and also ignore access control", but I still want to check it works correctly.

    @loadmodel(model='user', level=AccessType.READ, force=True, requiredFlags='my_key')